crates/cli/src/main.rs

```diff
@@ -0,0 +1,45 @@
+use beerus_core::client::BeerusClient;
+use beerus_core::config::Config;
+use beerus_rpc::BeerusRpc;
+use clap::Parser;
+use tracing::{error, info, Level};
+use tracing_subscriber::FmtSubscriber;
+
+#[derive(Parser, Debug)]
+#[command(version)]
+struct Args {
+    #[clap(short = 'c', long)]
+    conf: String,
+}
+
+#[async_std::main]
+async fn main() {
+    // env_logger::Builder::from_env(Env::default().default_filter_or("info")).init();
+    let subscriber = FmtSubscriber::builder().with_max_level(Level::INFO).finish();
+
+    tracing::subscriber::set_global_default(subscriber).expect("setting default subscriber failed");
+
+    let args = Args::parse();
+    let config = Config::from_file(&args.conf);
```

Comment:
todo from env?

Reply:
Good point, added a env option of the config flag isn't set. 
You think that UX is okay/understandable?

Reply:
If you look at the code - absolutely!
But if you blindly run it like `cargo run -p beerus-cli` without any `ENV` var , and see panic like 
```
thread 'main' panicked at /mount/beerus_new/crates/core/src/config.rs:163:14:
incorrect helios client config: execution client creation error: relative URL without a base
```
it may confuse imho

Reply:
what do you think would be a better way to handle?

Display the CLI usage if there is no config and no ENV vars set?

Reply:
look for config flag